libpromises/Makefile.am

```diff
@@ -204,9 +205,9 @@ BUILT_SOURCES = cf3lex.c cf3parse.h cf3parse.c \
 EXTRA_SCRIPTS = text2cstring.pl enterprise_extension.sed
 EXTRA_DIST = $(EXTRA_SCRIPTS)
 
-enterprise_extension.c: extensions_template.c enterprise_extension.sed
+enterprise_extension.c: enterprise_extension.c.pre enterprise_extension.sed
```

Comment:
It should be called ``extensions_template.c.pre``, because this is not only used for the enterprise extension, it's also used for the report collector plugin for cf-hub. We have two extensions.